### ISSUE TITLE
Multiplayer game list refreshing too quickly, sometimes you can click on the wrong game

### ISSUE THREAD

**Issue by RevoluPowered** [FAForever/client#567]
When clicking on the game list it's very easy to accidentally click on the wrong game, can you please investigate a better way of handling it?

**Comment by duk3luk3** [comment]
I agree. I think what we should do is disable refreshing while the mouse is in the list view.

We could also "stabilize" the list by never deleting items from it (only disabling games that can no longer be joined) and only appending items to the end. A full refresh of the list could be done when the tab is switched to, based on a button press, or when the mouse is not inside the list view.

**Comment by RevoluPowered** [comment]
Yeah sounds good. Would nice to see a simple refresh button instead in my opinion, and also a warning if the game has become full.

**Comment by duk3luk3** [comment]
PRs welcome ;)

I'm adding this to the "Next" milestone (i.e. "to be done soon") because it's a significant UX issue.

**Comment by RevoluPowered** [comment]
I'll check out the source code when I'm done at the end of today :)
